test-fixtures/ci/uri_update.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python
```

Comment:
Same as above

Reply:
removed it for simplicity since it won't work as intended below the license info.